projects/1.16-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json

```
@@ -0,0 +1,171 @@
+{
+  "entity.bosses_of_mass_destruction.comet": "彗星",
+  "entity.bosses_of_mass_destruction.lich": "暗夜巫妖",
+  "entity.bosses_of_mass_destruction.blue_fireball": "蓝色火球",
+  "entity.bosses_of_mass_destruction.soul_star": "灵魂之星",
+  "entity.bosses_of_mass_destruction.obsidilith" : "黑曜石柱",
+  "entity.bosses_of_mass_destruction.gauntlet" : "下界铁掌",
+  "entity.bosses_of_mass_destruction.void_blossom" : "虚空之花",
+  "entity.bosses_of_mass_destruction.spore_ball" : "孢子",
+  "entity.bosses_of_mass_destruction.petal_blade" : "花瓣利刃",
+
+  "block.bosses_of_mass_destruction.obsidilith_end_frame" : "黑曜石祭坛",
+  "block.bosses_of_mass_destruction.obsidilith_rune" : "黑曜石符文",
+  "block.bosses_of_mass_destruction.sealed_blackstone" : "尘封黑石砖",
+  "block.bosses_of_mass_destruction.gauntlet_blackstone" : "远古雕刻黑石",
+  "block.bosses_of_mass_destruction.chiseled_stone_altar" : "錾制石祭坛",
```

```suggestion
  "block.bosses_of_mass_destruction.chiseled_stone_altar" : "錾制石头祭坛",
```
原版的石xx已经在很早之前改为了石头xx